src/conf.py

```diff
@@ -12,9 +12,9 @@
 
 # -- Project information -----------------------------------------------------
 
-project = "Ferrocene Language Specification"
-copyright = "The Ferrocene Developers"
-author = "The Ferrocene Developers"
+project = "FLS"
+copyright = "The Ferrocene Developers and The Rust Project Contributors"
+author = "The Ferrocene Developers and The Rust Project Contributors"
```

Comment:
Do we just want to boil this down to "The Rust Project Contributors"?  At this point, given what's been done, I think we could consider the Ferrocene Developers to be Rust Project Contributors as well.

Reply:
Under the agreement of the transfer, we would leave existing copyright in place and add new copyright for any changes or updates. New files we would just put "The Rust Project Contributors".

Reply:
Yes... I had flagged this one because it's not a header on this file, it's a configuration that will apply this everywhere, to both old things as well as new ones.  My reading is that, due to the CLA and the wording of the Apache 2.0 license, which specifically defines:

> "Contributor" shall mean Licensor and any individual or Legal Entity on behalf of whom a Contribution has been received by Licensor and subsequently incorporated within the Work.

that all the existing authors would be "Rust Project Contributors" and so consolidating it would be OK for this cross-project purpose, and then on each file, we'd keep the individual copyright lines as you say.  But it's OK either way.